src/cf-expandables.less

```diff
@@ -697,7 +697,7 @@
 }
 
 .expandable-group .expandable_label {
-    .h4();
+    .heading-4();
     margin-bottom: 0;
```

Comment:
Ah, right. For some reason, had it in my head that all of the `heading-#` mixins didn't have any margins anymore.